tests/functional/config.js

```diff
@@ -20,8 +20,11 @@ let config = {
     census: [
       `${paths.test.wdioSpec}/census/*.spec.js`
     ],
+    components: [
+      `${paths.test.wdioSpec}/components/*/*.spec.js`
```

Comment:
Think we should just have them as their own spec without subfolders atm

Reply:
Once we add more components to the tests we will see how this will become when it comes to running specs etc. If it gets cumbersome will remove later on, for now will leave in